spec/openapi.yaml

```diff
@@ -10720,6 +10726,30 @@ paths:
   /translation-quality-api/v2/accounts/{accountUid}/profiles/{profileUid}/check-types/{checkTypeCode}/groups/{groupUid}/severity-level:
     $ref: './spec/translation_quality/check_type_severity_level_result.yaml#/x-paths/profiles_check_type_group_severity_level'
 
+  /translation-quality-api/v2/accounts/{accountUid}/profiles/{profileUid}/check-types/REGEX_CHECK/settings/rules:
+    $ref: './spec/translation_quality/regex_check_settings.yaml#/x-paths/profiles_regex_check_type_rules'
+
+  /translation-quality-api/v2/accounts/{accountUid}/profiles/{profileUid}/check-types/REGEX_CHECK/settings/rules/{ruleUid}:
+    $ref: './spec/translation_quality/regex_check_settings.yaml#/x-paths/profiles_regex_check_type_rule'
+
+  /translation-quality-api/v2/accounts/{accountUid}/profiles/{profileUid}/check-types/REGEX_CHECK/settings/rules/{ruleUid}/severity-level:
+    $ref: './spec/translation_quality/regex_check_settings.yaml#/x-paths/profiles_regex_check_type_rule_severity_level'
```

Comment:
If you add "makeResultForCertainFileFunction" for the file  it needs to change the file name to the regex_check_settings_result.yaml